src/ProcessManager.php

```diff
@@ -539,7 +539,14 @@ function ($carry, Slave $slave) {
 
         $conn->end(json_encode([
             'status' => $status,
-            'workers' => $this->slaveCount,
+            'workers' => [
+                'total' => count($this->slaves->getByStatus(Slave::ANY)),
```

Comment:
Would it make sense to move this into a `Slaves->getStatusSummary()` function?

Reply:
How's that?